config/install/globalredirect.settings.yml

```diff
@@ -1,9 +0,0 @@
-deslash: true
```

Comment:
let's keep the config without the global prefix for deslash and nonclean_to_clean